### PR TITLE
Fix KYC Photo Upload flow - II

### PR DESCRIPTION
**Pull request by kenan-mamedoff**

### CODE REVIEW

**Review by dangelovska — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/CameraHelper/KYCCameraViewController.swift [hunk 1]
```diff
@@ -257,6 +237,16 @@ class KYCCameraViewController: UIViewController, ViewProtocol {
         return view
     }()
     
+    private lazy var dismissButton: UIButton = {
+        let view = UIButton()
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/CameraHelper/KYCCameraViewController.swift:241]: we have our own button subclass

File: brd-ios/breadwallet/src/ApplicationController.swift [hunk 2]
```diff
@@ -144,6 +140,10 @@ class ApplicationController: Subscriber, Trackable {
     }
     
     func decideFlow() {
+        if let nvc = rootNavigationController {
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/ApplicationController.swift:143]: why now just use a coordinator who does this magic?

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift [hunk 3]
```diff
@@ -70,6 +69,8 @@ class KYCDocumentPickerViewController: BaseTableViewController<KYCCoordinator,
     // MARK: - KYCDocumentPickerResponseDisplay
     
     func displayTakePhoto(responseDisplay: KYCDocumentPickerModels.Photo.ResponseDisplay) {
+        LoadingView.hide()
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift:72]: not a singleton

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift [hunk 4]
```diff
@@ -78,15 +79,16 @@ class KYCDocumentPickerViewController: BaseTableViewController<KYCCoordinator,
     }
     
     func displayFinish(responseDisplay: KYCDocumentPickerModels.Finish.ResponseDisplay) {
+        LoadingView.hide()
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift:82]: same

File: brd-ios/breadwallet/src/ApplicationController.swift (hunk 2 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/ApplicationController.swift:143]: how?

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift (hunk 3 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift:72]: should it be?

File: brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift (hunk 4 above)
Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/Redesign/Scenes/KYC/KYCLevelTwo/DocumentPicker/KYCDocumentPickerViewController.swift:82]: should it be?

**Pull request merged**